documentation/source/conf.py

```diff
@@ -305,4 +307,21 @@
 }
 breathe_show_define_initializer = True
 
-autoclass_content = "both"
+# -- Extra setup for spelling check --------------------------------------------
+
+# Spelling check needs an additional module that is not installed by default.
+# Add it only if spelling check is requested so docs can be generated without it.
+
+if 'spelling' in sys.argv:
+    extensions.append("sphinxcontrib.spelling")
```

Comment:
I guess you can unconditionally enable it; it's part of the requirements.txt and should be available to everyone building the documentation.

Reply:
I suspect it fails right now, hence it not being enabled by default.

Reply:
I had copied that part of the setup from Django-CMS, without knowing exactly why they have it like that, but it seemed safe.